src/components/common/ForeignJurisdiction.vue

```diff
@@ -0,0 +1,116 @@
+<template>
+  <v-card flat class="pt-6 px-4">
+    <v-row no-gutters>
+      <v-col cols="12" sm="3">
+        <label class="title-label">Jurisdiction</label>
+      </v-col>
+      <v-col cols="12" sm="9">
+        <v-select
+          id="country-selector"
+          :items="countryNames"
+          v-model="selectedCountry"
+          filled
+          placeholder="Jurisdiction Country"
+          :rules="countryRules"
+          @change="emitChangedCountry($event)"
+        />
+        <v-select
+          v-if="canadaUsaRegions.length > 0"
+          id="region-selector"
+          :items="canadaUsaRegions"
+          v-model="selectedRegion"
+          filled
+          placeholder="Jurisdiction Province"
+          :rules="regionRules"
+          @change="emitChangedRegion($event)"
+        />
+      </v-col>
+    </v-row>
+  </v-card>
+</template>
+
+<script lang="ts">
+import { Component, Emit, Mixins } from 'vue-property-decorator'
+import { CountriesProvincesMixin } from '@/mixins'
+
+@Component({})
+export default class ForeignJurisdiction extends Mixins(CountriesProvincesMixin) {
+  selectedCountry = ''
+  selectedRegion = ''
+
+  /** Get country names as an array of strings. */
+  get countryNames (): string[] {
+    return this.getCountries().map(country => country.name)
```

Comment:
Very nice.

Reply:
Just one thing: I think the country code (and region code) needs to be provided to the Legal API, so you'll have to figure that out.

It may be possible to use an array of objects for a drop-down menu. Then you'd have access to all object properties when one is selected.